index.js

```diff
@@ -57,6 +57,63 @@ bot.registerCommand(
     }
 );
 
+bot.registerCommand(
+    "createPoll",
+    (msg, args) => {
+
+        const emojiSet = args[0];
+        let pollTitle = [];
+        let optionSet = [];
+        let optionBuilder = [];
+        let argCount = 0;
+
+        args.shift(); // Removes the first option that determins what emojis to use
+        args.push('99'); // Completely arbitrary number chosen to indicate the end of all the options. This is a bad idea and I need to figure out a different way to break up the options later.
```

Comment:
If you're going to use a sentinel value like this, a better one would be `Symbol`, like this:

```
const endOfOptions = new Symbol();
args.push(endOfOptions);
```

All `Symbol` instances are unique and only equal themselves, so they work really well for this.

That being said, modifying this array in place is probably not a good pattern, and it seems like you aren't even using the sentinel value anyways.